packages/@aws-cdk/aws-eks/lib/helm-chart.ts

```diff
@@ -53,6 +53,12 @@ export interface HelmChartOptions {
    * @default Duration.minutes(5)
    */
   readonly timeout?: Duration;
+
+  /**
+   * create namespace if not exist
+   * @default False
+   */
+  readonly createNamespace?: boolean;
```

Comment:
Sorry but this should be a separate PR :-) 

[think about the changelog entry :-)]

_I added comments on the namespace change as well here so it's PR can start from there_